SECURITY.md

```diff
@@ -130,3 +130,30 @@ work as a member of the Fastify Core team.
 * [__KaKa Ng__](https://github.com/climba03003)
 * [__James Sumners__](https://github.com/jsumners),
   <https://twitter.com/jsumners79>, <https://www.npmjs.com/~jsumners>
+
+## OpenSSF CII Best Practices
+
+[![CII Best Practices](https://bestpractices.coreinfrastructure.org/projects/7585/badge)](https://bestpractices.coreinfrastructure.org/projects/7585)
+
+There are three “tiers”: passing, silver, and gold.
+
+### Passing
+We meet 100% of the “passing” criteria.
+
+### Silver
+We meet 87% of the “silver” criteria. The gaps are as follows:
+  - we do not have a DCO or a CLA process for contributions.
```

Comment:
Isn't this it? https://github.com/fastify/fastify/blob/fc6bf5cb9f000ad4cda5e197f4fd22166a15dab9/CONTRIBUTING.md?plain=1#L136-L158

Reply:
ah, good point. I’m not sure if this counts or not, since there’s nothing automated enforcing a signoff or a CLA signing, but it might!

Reply:
(waiting for an answer from OpenJS legal)

Reply:
I heard back; unfortunately this doesn't count, as it needs to be per-commit or per-PR.

Thus the claim in this PR is sadly correct, and this should be good to merge.

Reply:
Per commit or per PR? That's rather nuts.

Reply:
I agree, which is why I haven't implemented this kind of check in any projects - since it puts undue burden on the contributor.

Reply:
Technically, it _is_ signed per PR (that no one reads) https://github.com/fastify/.github/blob/a0da6855b30a6918d6fbb0c80af3204a1c619a22/.github/PULL_REQUEST_TEMPLATE.md

Reply:
Is there CI or consistent human enforcement that that box is checked? If so, it would count, if not, I suspect not.

Reply:
No.

Reply:
Then for now I think this PR is mergeable as-is.

It'd be great to merge it soon, so I can close a number of OpenJS issues around it :-)